android/src/main/java/sqip/flutter/SquareInAppPaymentsFlutterPlugin.java

```diff
@@ -67,6 +77,48 @@ public void onMethodCall(MethodCall call, final Result result) {
       String currencyCode = call.argument("currencyCode");
       int priceStatus = call.argument("priceStatus");
       googlePayModule.requestGooglePayNonce(result, price, currencyCode, priceStatus);
+    } else if (call.method.equals("startCardEntryFlowWithBuyerVerification")) {
+      boolean collectPostalCode = call.argument("collectPostalCode");
+
+      HashMap<String, Object> moneyMap = call.argument("money");
+      Money money = new Money(
+        ((Integer)moneyMap.get("amount")).intValue(),
+        sqip.Currency.valueOf((String)moneyMap.get("currencyCode")));
```

Comment:
suggest to handle these data conversion inside the cardEntryModule, by putting them here make this module too big.

Reply:
Updated